values.yaml

```diff
@@ -590,7 +590,7 @@ registry:
   credentials:
     username: "harbor_registry_user"
     password: "harbor_registry_password"
-    # If using existingSecret, the key must be REGISTRY_PASSWD and REGISTRY_HTPASSWD
+    # Secret keys must be REGISTRY_PASSWD and REGISTRY_HTPASSWD. For key REGISTRY_PASSWD the value is the password. For key REGISTRY_HTPASSWD the value is the string in the password file generated by htpasswd where the username is harbor_registry_user and the encryption type is bcrypt. For example: `htpasswd -bBc passwordfile harbor_registry_user harbor_registry_password`. The username must be harbor_registry_user!
```

Comment:
Could you also update the description in README.md?

Reply:
Yes, I'll do that.

Reply:
I updated README.md. Lmk if you need anything else. Thanks!